packages/react-components/react-tags/stories/src/TagGroup/TagGroupSelect.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import {
+  TagGroup,
+  InteractionTag,
+  TagGroupProps,
+  InteractionTagPrimary,
+  InteractionTagSecondary,
+  Button,
+  makeStyles,
+  type TagValue,
+} from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '10px',
+  },
```

Comment:
_nit:_ Ditto for the `container` styles here on making these a `makeResetStyles` call.

Reply:
fixed